src/chrome/content/rules/Taobao.xml

```diff
@@ -1,25 +1,14 @@
 <!--
```

Comment:
Please use git revert, or create a new branch and cherry-pick the other commits. Removing whitespace in general is not helpful for readability.